setup.py

```diff
@@ -96,18 +96,16 @@ def build_extension(self, ext):
     keywords="molecular dynamics",
     ext_modules=ext_modules,
     packages=find_packages(),
-    python_requires=">=3.7",
+    python_requires=">=3.10",
     install_requires=[
-        "importlib-resources",
         "jax",
-        "jaxlib>0.3.13",
+        "jaxlib>0.4.6",
```

Comment:
nit: is there a known issue with 0.4.6 and before?

Reply:
No known issue, reverted change.